webui/react/src/components/DeterminedAuth.tsx

```diff
@@ -126,11 +115,9 @@ const DeterminedAuth: React.FC<Props> = ({ canceler }: Props) => {
         <p className={[css.errorMessage, css.message].join(' ')}>Incorrect username or password.</p>
       )}
       <Form.Item>
-        <ConfigProvider theme={buttonTheme}>
-          <Button disabled={!canSubmit} htmlType="submit" loading={isSubmitted} type="primary">
-            Sign In
-          </Button>
-        </ConfigProvider>
+        <Button disabled={!canSubmit} htmlType="submit" loading={isSubmitted} type="primary">
```

Comment:
can we use the themeprovider here?

Reply:
created https://hpe-aiatscale.atlassian.net/browse/WEB-1916, separate from Antd removal tasks